backend/src/main/java/de/neuefische/backend/repository/UserRepo.java

```diff
@@ -0,0 +1,33 @@
+package de.neuefische.backend.repository;
+
+import de.neuefische.backend.model.User;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public class UserRepo {
+    private final List<User> users;
+
+    public UserRepo(List<User> users) {
+        this.users = users;
+    }
+
+
+    public List<User> getUsers() {
+        return users;
+    }
+
+    public User addUser(User user) {
+        User newUser = new User(generateID(), user.name(), user.visitedCountries());
```

Comment:
I would generate the id in the service class. The Repo is normally only used for saving the object and not manipulating the objects.